ui/src/main/resources/FileManagerCode/DriveSheet.xml

```diff
@@ -2714,6 +2738,17 @@ require(['jquery', 'angular', 'JobRunner', 'angular-resource', 'angular-ui-boots
   padding: 2px;
 }
 
+#files .folder {
+  color: #777777;
+  font-size: 12px;
```

Comment:
Do you really need fixed font size? I usually prefer using relative values (e.g. smaller or some percent).